src/modules/dashboard/dashboard.js

```diff
@@ -47,6 +47,61 @@
             "yData": ["used", "20", "20", "35", "20", "20", "87", "14", "20", "25", "28", "44", "56", "78", "56", "67", "88", "76", "65", "87", "16"]
         };
 
+
+        /* Temporary mocked data for bar-chart */
+       
+        vm.chartTitleData = {
+            "title": "Pools",
+            "data": {
+                "total" : 21,
+                "error" : 3,
+                "warning" : 2
+            }
+        }
+
+        vm.chartData = [
```

Comment:
@kamleshverma1 Does the api will provide the similar response as you have mentioned here? And if we are going to make a separate call to get this data, we should have that function defined under utils.js and the mock json under api folder.